assets/prefabs/trunkRegion.prefab

```diff
@@ -0,0 +1,118 @@
+{
+    "DisplayName": {
+        "name": "Trunk"
+    },
+    "Trunk": {
+        "leftBlockFamily": "CoreBlocks:TrunkLeft",
+        "rightBlockFamily": "CoreBlocks:TrunkRight",
+        "openSound": "CoreAssets:DoorOpen",
+        "closeSound": "CoreAssets:DoorClose"
+    },
+    "Health": {
```

Comment:
Why are components like `Health` and `Trunk` in both `trunk.prefab` and `trunkRegion.prefab`?

Reply:
There was a lot of experimentation going on while trying to figure out which pieces of door and which pieces of chest needed to be combined and where. Health should only be required in the region, and Trunk should only be required in the item.